frontend/packages/local-storage-operator-plugin/src/plugin.ts

```diff
@@ -82,7 +75,6 @@ const plugin: Plugin<ConsumedExtensions> = [
     },
     flags: {
       required: [LSO_DEVICE_DISCOVERY],
```

Comment:
Here you are still keeping LSO_DEVICE_DISCOVERY flag But in all other places of the ceph-storage-plugin this is removed, is there any reason why you are still keeping it here?

Reply:
The flag LSO_DEVICE_DISCOVERY  means discovery CR present.
Discovery CR -> provides the disk list by discovering disks.
so its required always.

In ceph we were disallowing this flag LSO_DEVICE_DISCOVERY beacuse as mentioend above we want to override this disk list with OCS disk rep disk list

Reply:
ack